src/compiler-src.md

```diff
@@ -16,22 +19,11 @@ The repository consists of three main directories:
 
 - `compiler/` contains the source code for `rustc`. It consists of many crates
   that together make up the compiler.
-
```

Comment:
These blank lines add more space between the list elements, which I think makes this list easier to read. Can you re-add them?